### DISCUSSION TITLE
S40 Thermostat/HA Lennox30 IP Settings

### DISCUSSION THREAD

**Discussion by imcintyr** [PeteRager/lennoxs30#296] [Q&A]
Hi, forgive me if this is a newbie question, I am about 1wk into using HA/Lennox 30. 

We have the s40 thermostat and HA is connected to it locally. Is there a way to change the IP address Lennox30 is looking for the thermostat on? The other day I had to power cycle the furnace and the thermostat changed addresses from x.x.x.201 to x.x.x.132. I couldn't find a way to set the s40 to a static IP. Looks like the S40 only supports DHCP at least that I could find in the manual and a scouring of the settings pages. 

Is there simple way to edit that address? I read the README and looked around in the integration but nothing jumped out at me. Thanks for any help.

**Comment by PeteRager** [comment]
The simplest approach is to go into your router and set up a static dhcp reservation for the s40, this way the IP address will not change whenever the S40 reboots or loses power, etc.  How to do this varies based on the router manufacturer.  Do this and set it back to what is was originally, reboot the router or S40 and its IP address should go back to what it was originally and all should start working again.

**Comment by imcintyr** [comment]
Duh(n) and done! It has been so long since I have done that. Usually setting up a static IP is my go-to. Thanks!
